io.github.teamclouday.android-mic.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id": "io.github.teamclouday.android-mic",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "android-mic",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=network",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro",
+    "--device=usb",
```

Comment:
device=usb isn't allowed as it is not compatible with most distro.

Reply:
i disable the usb option. I'm sure upstream will be fine with that, he don't really care about flatpak

Reply:
applications that need access to USB should use `device=all`.

don't know what not working since you are not telling.

Reply:
It was an error about accessory on the phone. I think the code for the usb option is not perfect currently, so it's fine to disable it for now utils it get fixed. I'm not even sure the error was due to flatpak, but more about detecting all phones

Reply:
how do you access the device?

Reply:
so basically you don't know. You are just winging it.

Reply:
The doc that would have answered all the questions https://docs.flatpak.org/en/latest/sandbox-permissions.html#device-access

Reply:
Fyi, i already tested with `device=all` and it didn't work either. The adb one worked with no `device=all` or `device=usb` for some reason.

> If the app is not functional there is no reason to submit it.

Ofc, but in this case, it's just a feature of the app that don't work. Plus, the usb option can easilly be replaced by adb, so it not really important